Automattic/jetpack
# [Boost] Fix features/optimizations initialization

## Proposed changes:

* This update fixes features and optimizations not getting their setup_trigger respected.
* All of them had `init`, but they were actually getting initialized on `plugins_loaded`, which is where the main Optimizations class [gets initialized](https://github.com/Automattic/jetpack/blob/c37aca77f5ed00429b428154ecd12ba6a8a88bba/projects/plugins/boost/app/features/optimizations/Optimizations.php#L138).
* To preserve backwards compatibility, their setup_trigger has been updated to `plugins_loaded`. 

### Other information:


## Jetpack product discussion



p1675866389883209-slack-C016BBAFHHS

## Does this pull request change what data or activity we track or use?



No.

## Testing instructions:






1. Install this version of the plugin;
2. Activate features;
3. Make sure everything works as expected.
